wgsl/index.bs

```diff
@@ -8581,20 +8581,33 @@ The interface includes:
     * [=Texture resources=]
     * [=Sampler resources=]
 
-When an [=identifier=] in a [=function body=] [=resolves=] to a [=module scope|module-scope=] [[#var-and-value|variable or value declaration]],
-then we say that variable or value is <dfn>statically accessed</dfn> by the function.
-Note that being statically accessed is independent of whether an execution of the shader
-will actually evaluate the expression referring to the variable,
-or even execute the statement that may enclose the expression.
+A declaration *D* is <dfn>statically accessed</dfn> by a shader when:
+* An identifier [=resolving=] to *D* appears in the [=function declaration|declaration=]
+    of any of the [=functions in a shader stage|functions in the shader stage=].
+* An identifier [=resolving=] to *D* is used to define a type for a [=statically accessed=] declaration.
+* An identifier [=resolving=] to *D* is used in the initializer for a [=statically accessed=] declaration.
+* An identifier [=resolving=] to *D* is by an attribute used by a [=statically accessed=] declaration.
+
+<div class="note">Note: Static access is recursively defined, taking into account the following:
+* All the parts of a function declaration including attributes, formal parameters, return type, and function body
```

Comment:
Nit: Missing a period at the end of this bullet.

Reply:
done